about.html

```diff
@@ -0,0 +1,216 @@
+              class="testimonials-background"
+              src="images/sandrine.jpg"
+              alt="2020 Testimonial by Sandrine Lebon"
+            />
+            <h2 class="testimonials-title">Sandrine Lebon</h2>
+            <p class="testimonials-info">
+              Bibi Af is very professional. She makes you feel at ease and her
+              technique is great. She is really good at getting rid of those
+              knots.
+            </p>
+          </div>
+          <div class="testimonials-box box2">
+            <img
+              class="testimonials-background"
+              src="images/michelle.jpg"
+              alt="Testimonial by Isabelle Goetz"
+            />
+            <h2 class="testimonials-title">Isabelle Goetz</h2>
+            <p class="testimonials-info">
+              Home service is so convenient for me as I've just had a baby
+              recently. I looked online and found Af Esthétique. I give Bibi Af
+              a rating of 5 stars. She's the best I've known so far.
+            </p>
+          </div>
+        </div>
+      </section>
+      <footer>
+        <div class="customers-container">
+          <h2 class="customers">Satisfied Makeup Customers</h2>
+          <div class="bar customers-bar"></div>
+          <div class="customers-small-container">
+            <div class="customers-img-container">
+              <img
+                src="images/customer.jpg"
+                class="customers-img"
+                alt="Satisfied customer 1"
+              />
+            </div>
+            <div class="customers-img-container">
+              <img
+                src="images/customer2.jpg"
+                class="customers-img"
+                alt="Satisfied customer 2"
+              />
+            </div>
+            <div class="customers-img-container">
+              <img
+                src="images/customer3.jpg"
+                class="customers-img"
+                alt="Satisfied customer 3"
+              />
+            </div>
+            <div class="customers-img-container">
+              <img
+                src="images/customer4.jpg"
+                class="customers-img"
+                alt="Satisfied customer 4"
+              />
+            </div>
+            <div class="customers-img-container">
+              <img
+                src="images/customer5.jpg"
+                class="customers-img"
+                alt="Satisfied customer 5"
+              />
+            </div>
+          </div>
+        </div>
+        <div class="footer-container-about">
+          <div class="footer-content">
+            <img src="images/logo.png" class="footer-img" alt="logo" />
+            <p>
+              We are delighted to be welcoming more women across Mauritius into
+              our aesthetic community. We are now offering our home treatments
+              mainly in the North, with more areas coming soon.
+            </p>
+          </div>
+        </div>
```

Comment:
- [ ] This footer should be white in color in mobile version so that the section could easily seperable from upper section. Also your design is not following the behance design, I welcome you to fix this as well.

### Mobile version

### Behance design

![Screenshot from 2021-07-09 21-07-37](https://user-images.githubusercontent.com/22434660/125103949-78ea5100-e0fa-11eb-810e-e7f436c255a0.png)


### Your design

![Screenshot from 2021-07-09 21-10-21](https://user-images.githubusercontent.com/22434660/125103932-74259d00-e0fa-11eb-8e97-79459c6ad1ae.png)